br_account_payment/models/account_payment.py

```diff
@@ -21,13 +21,6 @@ def default_get(self, fields):
                 'default_amount', rec.get('amount', 0.0))
         return rec
 
-    def action_validate_invoice_payment(self):
```

Comment:
O método se faz necessário quando o pagamento for criado a partir da da view 'Contas a Receber e Pagar'.